src/js/components/Calendar/StyledCalendar.js

```diff
@@ -61,35 +73,50 @@ const slideStyle = props => {
   `;
 };
 
+const weeksSizeStyle = props => {
+  return css`
+    ${props.fillContainer && 'display: flex;'}
+    ${props.fillContainer && 'flex-direction: column;'}
+    ${props.fillContainer && 'height: 100%;'}
+  `;
+};
 const StyledWeeks = styled.div`
   position: relative;
+  ${props => weeksSizeStyle(props)}
   ${props => props.slide && slideStyle(props)};
 `;
 
 StyledWeeks.defaultProps = {};
 Object.setPrototypeOf(StyledWeeks.defaultProps, defaultProps);
 
+const weekSizeStyle = props => {
+  return css`
+    display: flex;
+    justify-content: space-between;
+    ${props.fillContainer && 'flex: 1;'}
+  `;
+};
 const StyledWeek = styled.div`
-  display: flex;
-  flex-direction: row;
-  flex-justify: between;
+  ${props => weekSizeStyle(props)}
```

Comment:
Can we keep `display` and `justify-content` here and then: `${props => props.fillContainer && 'flex: 1;'}`?

Reply:
Sounds good. Update made.